src/webview_provider/insightsPanel.ts

```diff
@@ -118,6 +121,133 @@ export class InsightsPanel extends AltimateWebviewProvider {
     }
   }
 
+  private async fetchProjectIntegrations(syncRequestId: string | undefined) {
+    try {
+      console.log("Fetching project integrations");
+      const response = await this.altimateRequest.fetchProjectIntegrations();
+
+      if (syncRequestId) {
+        this._panel!.webview.postMessage({
+          command: "response",
+          args: {
+            syncRequestId,
+            body: response,
+            status: true,
+          },
+        });
+      }
+    } catch (err) {
+      console.info(
+        "could not fetch project integrations",
+        (err as Error).message,
+      );
+      this._panel!.webview.postMessage({
+        command: "response",
+        args: {
+          syncRequestId,
+          body: {
+            response: [],
+          },
+          status: false,
+        },
+      });
+    }
+  }
+
+  async downloadManifestLocally(
```

Comment:
move this function to `src/altimate.ts` for downloading files. So that it can be used in other places.